pyclesperanto_prototype/_tier9/_statistics_of_labelled_pixels.py

```diff
@@ -276,6 +276,11 @@ def statistics_of_labelled_pixels(intensity_image : Image = None, label_image :
     power(sum_dim, result_vector, 0.5)
     region_props['standard_deviation_intensity'] = pull(result_vector)[0]
 
+    # standard error intensity
+    crop(sum_per_label, sum_dim, measurements_start_x, 3, 0) # number of pixels/voxels = area
```

Comment:
Ok, wow. I must admit, I don't get this. How about using `region_props['area']` ?

Reply:
This is area calculation copied and pasted from line 199 (the same also appears on line 260) above.
I am assuming this leaves the area in the GPU memory, so that I can then use `power()` to calculate the square root, rather than importing np.sqrt or math.sqrt just for this.

Reply:
How would you calculate the square root of `region_props['area']`?

Reply:
`np.sqrt(region_props['area'])`? (I'm speculating)

Reply:
is this better :-)

actually that was my initial approach, but then I saw that you used power() everywhere else, so I tried to follow your coding style... ;-)